controller/persistence/api_session_store.go

```diff
@@ -109,6 +116,48 @@ type apiSessionStoreImpl struct {
 	symbolIdentity boltz.EntitySymbol
 }
 
+func (store *apiSessionStoreImpl) onDeleteToEventual(i ...interface{}) {
```

Comment:
Change to execute as `DeleteById()`  and pass a tx to `EventualEventer.AddEvent(tx, name, []byte(id))`